gum/gumscript-runtime-core.js

```diff
@@ -6,6 +6,8 @@
         dispatcher = new MessageDispatcher();
     };
 
+    var longSize = Process.arch === 'x64' || Process.platform === 'windows' ? 64 : 32;
```

Comment:
It would be good to use ES6 here and say:
`const longSize = ...`
I suspect the condition should be like this though:
```JavaScript
const longSize = (Process.pointerSize == 8 && Process.platform !== 'windows') ? 64 : 32;
```